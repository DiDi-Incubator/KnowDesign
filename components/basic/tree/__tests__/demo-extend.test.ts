import { extendTest } from '../../../tests/shared/demoTest';

extendTest('tree', { skip: ['big-data.md', 'virtual-scroll.md'] });
