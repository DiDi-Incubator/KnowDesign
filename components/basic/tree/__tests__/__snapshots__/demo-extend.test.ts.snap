// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ./components/basic/tree/demo/basic.md extend context correctly 1`] = `
LoadedCheerio {
  "0": Document {
    "children": Array [],
    "endIndex": null,
    "next": null,
    "parent": null,
    "prev": null,
    "startIndex": null,
    "type": "root",
  },
  "_root": LoadedCheerio {
    "0": Document {
      "children": Array [],
      "endIndex": null,
      "next": null,
      "parent": null,
      "prev": null,
      "startIndex": null,
      "type": "root",
    },
    "_root": [Circular],
    "length": 1,
    "options": Object {
      "decodeEntities": true,
      "xml": false,
    },
  },
  "length": 1,
  "options": Object {
    "decodeEntities": true,
    "xml": false,
  },
}
`;

exports[`renders ./components/basic/tree/demo/basic-controlled.md extend context correctly 1`] = `
LoadedCheerio {
  "0": Document {
    "children": Array [],
    "endIndex": null,
    "next": null,
    "parent": null,
    "prev": null,
    "startIndex": null,
    "type": "root",
  },
  "_root": LoadedCheerio {
    "0": Document {
      "children": Array [],
      "endIndex": null,
      "next": null,
      "parent": null,
      "prev": null,
      "startIndex": null,
      "type": "root",
    },
    "_root": [Circular],
    "length": 1,
    "options": Object {
      "decodeEntities": true,
      "xml": false,
    },
  },
  "length": 1,
  "options": Object {
    "decodeEntities": true,
    "xml": false,
  },
}
`;

exports[`renders ./components/basic/tree/demo/customized-icon.md extend context correctly 1`] = `
LoadedCheerio {
  "0": Document {
    "children": Array [],
    "endIndex": null,
    "next": null,
    "parent": null,
    "prev": null,
    "startIndex": null,
    "type": "root",
  },
  "_root": LoadedCheerio {
    "0": Document {
      "children": Array [],
      "endIndex": null,
      "next": null,
      "parent": null,
      "prev": null,
      "startIndex": null,
      "type": "root",
    },
    "_root": [Circular],
    "length": 1,
    "options": Object {
      "decodeEntities": true,
      "xml": false,
    },
  },
  "length": 1,
  "options": Object {
    "decodeEntities": true,
    "xml": false,
  },
}
`;

exports[`renders ./components/basic/tree/demo/directory.md extend context correctly 1`] = `
LoadedCheerio {
  "0": Document {
    "children": Array [],
    "endIndex": null,
    "next": null,
    "parent": null,
    "prev": null,
    "startIndex": null,
    "type": "root",
  },
  "_root": LoadedCheerio {
    "0": Document {
      "children": Array [],
      "endIndex": null,
      "next": null,
      "parent": null,
      "prev": null,
      "startIndex": null,
      "type": "root",
    },
    "_root": [Circular],
    "length": 1,
    "options": Object {
      "decodeEntities": true,
      "xml": false,
    },
  },
  "length": 1,
  "options": Object {
    "decodeEntities": true,
    "xml": false,
  },
}
`;

exports[`renders ./components/basic/tree/demo/drag-debug.md extend context correctly 1`] = `
LoadedCheerio {
  "0": Document {
    "children": Array [],
    "endIndex": null,
    "next": null,
    "parent": null,
    "prev": null,
    "startIndex": null,
    "type": "root",
  },
  "_root": LoadedCheerio {
    "0": Document {
      "children": Array [],
      "endIndex": null,
      "next": null,
      "parent": null,
      "prev": null,
      "startIndex": null,
      "type": "root",
    },
    "_root": [Circular],
    "length": 1,
    "options": Object {
      "decodeEntities": true,
      "xml": false,
    },
  },
  "length": 1,
  "options": Object {
    "decodeEntities": true,
    "xml": false,
  },
}
`;

exports[`renders ./components/basic/tree/demo/draggable.md extend context correctly 1`] = `
LoadedCheerio {
  "0": Document {
    "children": Array [],
    "endIndex": null,
    "next": null,
    "parent": null,
    "prev": null,
    "startIndex": null,
    "type": "root",
  },
  "_root": LoadedCheerio {
    "0": Document {
      "children": Array [],
      "endIndex": null,
      "next": null,
      "parent": null,
      "prev": null,
      "startIndex": null,
      "type": "root",
    },
    "_root": [Circular],
    "length": 1,
    "options": Object {
      "decodeEntities": true,
      "xml": false,
    },
  },
  "length": 1,
  "options": Object {
    "decodeEntities": true,
    "xml": false,
  },
}
`;

exports[`renders ./components/basic/tree/demo/dynamic.md extend context correctly 1`] = `
LoadedCheerio {
  "0": Document {
    "children": Array [],
    "endIndex": null,
    "next": null,
    "parent": null,
    "prev": null,
    "startIndex": null,
    "type": "root",
  },
  "_root": LoadedCheerio {
    "0": Document {
      "children": Array [],
      "endIndex": null,
      "next": null,
      "parent": null,
      "prev": null,
      "startIndex": null,
      "type": "root",
    },
    "_root": [Circular],
    "length": 1,
    "options": Object {
      "decodeEntities": true,
      "xml": false,
    },
  },
  "length": 1,
  "options": Object {
    "decodeEntities": true,
    "xml": false,
  },
}
`;

exports[`renders ./components/basic/tree/demo/line.md extend context correctly 1`] = `
LoadedCheerio {
  "0": Document {
    "children": Array [],
    "endIndex": null,
    "next": null,
    "parent": null,
    "prev": null,
    "startIndex": null,
    "type": "root",
  },
  "_root": LoadedCheerio {
    "0": Document {
      "children": Array [],
      "endIndex": null,
      "next": null,
      "parent": null,
      "prev": null,
      "startIndex": null,
      "type": "root",
    },
    "_root": [Circular],
    "length": 1,
    "options": Object {
      "decodeEntities": true,
      "xml": false,
    },
  },
  "length": 1,
  "options": Object {
    "decodeEntities": true,
    "xml": false,
  },
}
`;

exports[`renders ./components/basic/tree/demo/search.md extend context correctly 1`] = `
LoadedCheerio {
  "0": Document {
    "children": Array [],
    "endIndex": null,
    "next": null,
    "parent": null,
    "prev": null,
    "startIndex": null,
    "type": "root",
  },
  "_root": LoadedCheerio {
    "0": Document {
      "children": Array [],
      "endIndex": null,
      "next": null,
      "parent": null,
      "prev": null,
      "startIndex": null,
      "type": "root",
    },
    "_root": [Circular],
    "length": 1,
    "options": Object {
      "decodeEntities": true,
      "xml": false,
    },
  },
  "length": 1,
  "options": Object {
    "decodeEntities": true,
    "xml": false,
  },
}
`;

exports[`renders ./components/basic/tree/demo/switcher-icon.md extend context correctly 1`] = `
LoadedCheerio {
  "0": Document {
    "children": Array [],
    "endIndex": null,
    "next": null,
    "parent": null,
    "prev": null,
    "startIndex": null,
    "type": "root",
  },
  "_root": LoadedCheerio {
    "0": Document {
      "children": Array [],
      "endIndex": null,
      "next": null,
      "parent": null,
      "prev": null,
      "startIndex": null,
      "type": "root",
    },
    "_root": [Circular],
    "length": 1,
    "options": Object {
      "decodeEntities": true,
      "xml": false,
    },
  },
  "length": 1,
  "options": Object {
    "decodeEntities": true,
    "xml": false,
  },
}
`;
